### ISSUE TITLE
Latest publish missing tags

### ISSUE THREAD

**Issue by ddelange** [stevearc/pypicloud-docker#38]
Hi @stevearc 👋 

It [looks](https://github.com/stevearc/pypicloud-docker/actions/runs/3804581932) like the latest deploy only pushed `latest`, and [failed to push `1.3.12` tags](https://hub.docker.com/r/stevearc/pypicloud/tags?page=1&name=1.3.12).

Any idea what might have happened?

**Comment by stevearc** [comment]
Not sure, let me try again

**Comment by stevearc** [comment]
[built](https://hub.docker.com/layers/stevearc/pypicloud/1.3.12/images/sha256-ab1538a233988973fc0056b05c2d86b50b3bce7a014ea4ed94e4a8e5ae4e45d3?context=repo)!

**Comment by ddelange** [comment]
awesome that was quick 🤩
